roles/debian/pam_ldap/templates/ldap.conf.j2

```diff
@@ -1,9 +1,11 @@
 BASE {{ pam_ldap.lookup_base }}
-URI	{{ pam_ldap.endpoints | join(' ') }}
-{% if pam_ldap.ssl_certificate is defined and pam_ldap.ssl_certificate %}
+URI {{ pam_ldap.endpoints | join(' ') }}
+{% if pam_ldap.ssl_certificate is defined and pam_ldap.ssl_use_system_ca %}
```

Comment:
Get rid of `pam_ldap.ssl_certificate is defined and ` here - no need to insist on an empty `pam_ldap.ssl_certificate` variable to use the system CA certs.